spec/unit/shoulda/matchers/action_controller/permit_matcher_spec.rb

```diff
@@ -180,6 +180,16 @@ def params_with_conditional_require(params, *filters)
       for(:show, verb: :get, params: { slug: 'foo' })
   end
 
+  it 'works when #fetch is used instead of #require (issue #495)' do
+    define_controller_with_strong_parameters(action: :create) do
+      params.fetch(:order, {}).permit(:eta, :diner_id)
+    end
+
+    expect(controller).
+      to permit(:eta, :diner_id).
+      for(:create, params: { order: { foo: '' } })
```

Comment:
Yeah.. Ideally that shouldn't need to happen for this, but I'm not sure why it's needed. I get why it's needed when using `require`, but not `fetch`.